src/photos/ducks/timeline-clusters/dates.js

```diff
@@ -53,7 +54,7 @@ export const isSameMonth = (f, newerDate, olderDate) => {
 export const isSameDay = (f, newerDate, olderDate) => {
   const newer = formatDate(f, newerDate)
   const older = formatDate(f, olderDate)
-  return differenceInCalendarDays(newer, older) < 1
+  return differenceInCalendarDays(newer, older) === 0
```

Comment:
Should we do the same modification with isSameHour / isSameMonth ? Why doing this only on sameDay ?

Reply:
Right!